angular.json

```diff
@@ -1,30 +1,36 @@
 {
   "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "cli": {
+    "analytics": false
```

Comment:
package manager yarn?

Reply:
there is no need to tell the package manager now. yarn and npm, anyone from 2 can be used as a dependency manager by default.